### PR TITLE
Add default sub-team reviewers

### PR DESCRIPTION
**Pull request by liangxia**
Add some default reviewers based on script author in the sub-dir.

### CODE REVIEW

File: features/logging/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+approvers:
+
+reviewers:
+- QiaolingTang
```
Comment (pruan-rht) [inline features/logging/OWNERS]: @liangxia Please add @anpingli  as a reviewers for logging

File: features/svc-catalog_asb/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+approvers:
+
+reviewers:
+- chengzhang1016
+- jianzhangbjz
```
Comment (jianzhangbjz) [inline features/svc-catalog_asb/OWNERS]: Could you help add @emmajiafan in here? Thanks!

File: features/web/service-catalog/OWNERS [hunk 3]
```diff
@@ -0,0 +1,4 @@
+approvers:
+
+reviewers:
+- SSshahan
```
Comment (jianzhangbjz) [inline features/web/service-catalog/OWNERS]: Could you help add me and @emmajiafan in here? Thanks!

File: features/logging/OWNERS (hunk 1 above)
Comment (liangxia) [reply features/logging/OWNERS]: @pruan-rht  Added.

File: features/svc-catalog_asb/OWNERS (hunk 2 above)
Comment (liangxia) [reply features/svc-catalog_asb/OWNERS]: @jianzhangbjz   @emmajiafan Added.

File: features/admin/OWNERS [hunk 4]
```diff
@@ -0,0 +1,6 @@
+approvers:
+
+reviewers:
+- chengzhang1016
+- pruan-rht
+- zhouying7780
```
Comment (xingxingxia) [inline features/admin/OWNERS]: Suggest remove yinzhou under admin, remove me from cli (keep yinzhou) and add me under admin

Comment (liangxia) [reply features/admin/OWNERS]: @xingxingxia Updated.

**Comment by liangxia** [comment]
Any more comments ?
Or can we get this merged first so there is a basic functional one, and update later when necessary ?

**Comment by pruan-rht** [comment]
This is a good start, we can amend it more later if needed

**Comment by pruan-rht** [comment]
This is a good start, we can amend it more later if needed

**Pull request merged**